.merlin.in

```diff
@@ -40,6 +40,8 @@ S API
 B API
 S ide
 B ide
+S gramlib__pack
+B gramlib__pack
```

Comment:
Are you sure we want this? This way if I jump to definition I end up on a file I should not edit.